src/controller/api/index.js

```diff
@@ -0,0 +1,75 @@
+const {
+  getNotesList,
+  validKeys,
+  writeNotesToFile,
+  identifyNoteId,
+} = require("../../utility/util");
+const { v4: uuidv4 } = require("uuid");
+
+const getNotes = (req, res) => {
+  // get notes data fom the db file by reading, then parse the string and then res.json
+
+  const notesList = getNotesList();
+
+  res.json(notesList);
+};
+
+const addNewNote = (req, res) => {
+  const payLoad = req.body;
+
+  const isValid = validKeys.every((each) => {
+    return Object.keys(payLoad).includes(each);
+  });
+
+  if (isValid) {
```

Comment:
We haven't got a catch here for if the note isn't valid. Make sure you cover all eventualities.